src/utilities/suggestionList.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
```

Comment:
this file should go in the jsutils folder - that is where we keep the dependency-free JS utilities that are not specific to graphql, just happen to be used here.